src/mlir/generator.cc

```diff
@@ -329,7 +337,11 @@ namespace mlir::verona
     if (isLocalRef(ast))
     {
       // We use allocas to track location and load/stores to track access
-      auto var = symbolTable.lookup(getTokenValue(ast));
+      auto name = getTokenValue(ast);
+      auto var = symbolTable.lookup(name);
+      if (!var)
+        return parsingError(
```

Comment:
Not sure it matters at the moment, but this is probably the kind of error that shouldn’t stop parsing (and suppressing future errors). For the bootstrap compiler, the input should always have been type checked with the second compiler, so it might not matter...

Reply:
That is true, but I didn't want to have a potential error unchecked (and null pointers dangling). I could just change that to an assert, as the front-end should really have checked it.